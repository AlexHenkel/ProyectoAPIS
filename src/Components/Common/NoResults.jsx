import React from 'react'
import styled from 'styled-components'
import { Typography as OriginalTypography } from 'material-ui'

const Typography = styled(OriginalTypography)`
  margin-top: 50px !important;
  margin-bottom: 50px !important;
`

export default () => (
  <Typography type="subheading" color="primary" gutterBottom align="center">
    No se encontraron resultados <span>:(</span>
  </Typography>
)
