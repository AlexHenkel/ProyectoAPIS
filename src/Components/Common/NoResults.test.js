import React from 'react'
import NoResults from './NoResults'

describe('NoResults component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoResults />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
