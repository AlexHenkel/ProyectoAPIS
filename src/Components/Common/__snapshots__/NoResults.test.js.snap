// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component renders correctly 1`] = `
<h3
  className="MuiTypography-root-1 MuiTypography-subheading-8 MuiTypography-colorPrimary-21 MuiTypography-gutterBottom-18 MuiTypography-alignCenter-14 sc-bdVaJa dEnwzp"
>
  No se encontraron resultados 
  <span>
    :(
  </span>
</h3>
`;
